File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        int res = 0;
        unordered_set<int> store(nums.begin(), nums.end());

        int ans = 0;

        for(int num: nums){
            int length = 0, cur = num;
            while(store.find(cur)!= store.end()){
                length++;
                cur++;
            }
            ans = max(ans, length);
        }
        return ans;
    }
};

File: Data Structures & Algorithms/maximum-subarray/submission-0.py
class Solution:
    def maxSubArray(self, nums: List[int]) -> int:
        if len(nums) == 1:
            return nums[0]
        pre = 0
        res = nums[0]
        for n in nums:
            res = max(res, pre + n)
            pre = max(0, pre + n)
        return res
